### ISSUE TITLE
Cannot disable mgmt ip, when deployed through module

### ISSUE THREAD

**Issue by madhukar32** [f5devcentral/terraform-aws-bigip-module#56]
If you look at the example below we are setting the `mgmt_eip` var to false deliberately and `mgmt_subnet_ids`  object also set to `public_ip` as false

```hcl
module bigip-1 {
  source                              = "../../../modules/terraform-aws-bigip-module"
  prefix                                = format("%s-1", var.aws_name_prefix)
  ec2_key_name                = aws_key_pair.ssh.key_name
  mgmt_subnet_ids           = local.is_multinode_bigip ? local.mgmt_subnet_ids : [element(local.mgmt_subnet_ids, 0)]
  mgmt_securitygroup_ids   = [aws_security_group.management.id]
  external_subnet_ids         = local.is_multinode_bigip ? local.external_subnet_ids : [element(local.external_subnet_ids, 0)]
  external_securitygroup_ids  = [aws_security_group.external.id]
  internal_subnet_ids         = local.is_multinode_bigip ? local.internal_subnet_ids : [element(local.internal_subnet_ids, 0)]
  internal_securitygroup_ids  = [aws_security_group.internal.id]
  mgmt_eip                    = false
  f5_username                 = var.admin_username
  f5_password                 = var.admin_password
}

locals {
mgmt_subnet_ids = [for idx, device in var.devices: {
    "subnet_id" = device.public_subnet_id
    "public_ip" = false
    "private_ip_primary" =  ""
  }]
}
```

**Comment by trinaths** [comment]
Created JIRA INFRAANO-488 for internal tracking

**Comment by papineni87** [comment]
@madhukar32 can you try the scenario with the latest code in master branch ? i had pushed a fix for it

**Comment by papineni87** [comment]
Issue is fixed in v0.9.6, please reopen the issue if you are still facing it.
